smart-py/objkt_swap.py

```diff
@@ -17,135 +38,679 @@ def __init__(self, objkt, hdao, manager, metadata, curate):
             genesis = sp.timestamp(0),
             curate = curate,
             locked = False
-            )
-    
+        )
+
     @sp.entry_point
     def genesis(self):
-        sp.verify((sp.sender == self.data.manager) & ~(self.data.locked))
+        sp.verify(
+            (
+                sp.sender == self.data.manager
+            ) & ~(
+                self.data.locked
+            )
+        )
+
+        # what's up with adding 45 days?
```

Comment:
That seems to be date after which no more hDAOs were distributed. I guess this entry point is not necessary anymore?

Reply:
it looks like there might be a few that can be retired